src/components/form/Form.vue

```diff
@@ -567,14 +567,22 @@ export default {
         },
         getHiddenFields() {
             const hiddenInputFields = this.$el.querySelectorAll('input[type=hidden]');
-            const fieldData = {
-            };
 
-            hiddenInputFields.forEach((field) => {
-                fieldData[field.name] = field.value;
-            });
+            return [...hiddenInputFields].map(this.parseBooleanStringsFromInputField);
+        },
+        parseBooleanStringsFromInputField(inputField) {
```

Comment:
Methods all need a comment to explain the function, we aren't using JSdocs at the moment so just a short comment to explain what its/what it returns will do